assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -288,6 +300,9 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
         if (isResizingFromCorner !== newResizingMode) {
           setIsResizingFromCorner(newResizingMode);
         }
+        if (isEditMode) {
+          setActionHappening(true);
```

Comment:
This is needed here since otherwise, if we change the height of the element in the edit mode via changing the text (e.g. by adding some lines), the element will be out of sync and Moveable will not work as expected.

Not sure if there's a better alternative for this.

Reply:
Perhaps an alternative could be tracking the original height within the Text Edit mode and updating the properties of the Text element every time when there's a difference with the previous height? This way it would always be up to date for Moveable.

This would cause a new history entry for each edit which causes a height change though, also not the best option.

Reply:
I believe the movable only tracks the outline of the target DOM element. So all we need to do to keep them in sync is to call `moveable.updateRect()`. It's already called in the `SingleSelectionMovable`, but since it's outside of the rendering path it's probably doesn't get called. So we can either call `updateRect()` manually, or we could find a way to get the whole `SingleSelectionMovable` to rerender.

Reply:
It was actually the `updateForResizeEvent` used for the text element which didn't work as expected unless the element was updated.

Additionally, it might make sense to update the element before each resizing for adding the text editing that happened before resizing as a separate history entry. Thoughts?

Without updating the element properties before doing resizing, the following happened:
1. In edit mode, I added/removed a few extra lines
2. Since `updateRect()` is happening in the text edit mode, the moveable follows the height change correctly.
3. Now when I resize the element, the inner content and the Moveable go out of sync.

Example:
![resizing-edit](https://user-images.githubusercontent.com/3294597/85836022-43482380-b796-11ea-8ffd-0ecdbabb4f38.gif)

This doesn't happen if the element properties are updated before resizing (as currently via the `actionsHappening`). Thoughts?

Reply:
Using `isAnythingTransforming` now.